roles/pulp_content_guard/tasks/main.yml

```diff
@@ -12,3 +12,6 @@
   with_items: "{{ pulp_content_guard_x509_cert_guards }}"
   loop_control:
     label: "{{ item.name }}"
+
+- name: Ensure RBAC cert guards exist
+  include_tasks: rbac/rbac.yml
```

Comment:
nit: for a static task include with no condition or loop, you can use `import_tasks`. `include_tasks` is an actual task that is evaluated at runtime, and has a small overhead.